atd-vzd/schema/atd_txdot_crashes.sql

```diff
@@ -219,84 +216,215 @@ CREATE TABLE public.atd_txdot_crashes (
     address_confirmed_secondary text,
     est_comp_cost numeric(10,2) DEFAULT 0.00,
     est_econ_cost numeric(10,2) DEFAULT 0.00,
-    "position" public.geometry,
+    "position" public.geometry(Geometry,4326),
     apd_confirmed_fatality character varying(1) DEFAULT 'N'::character varying NOT NULL,
     apd_confirmed_death_count integer,
-    micromobility_device_flag character varying(1) DEFAULT 'N'::character varying NOT NULL
+    micromobility_device_flag character varying(1) DEFAULT 'N'::character varying NOT NULL,
+    cr3_stored_flag character varying(1) DEFAULT 'N'::character varying NOT NULL,
+    apd_human_update character varying DEFAULT 'N'::character varying NOT NULL,
+    speed_mgmt_points numeric(10,2) DEFAULT 0.00,
+    geocode_match_quality numeric,
+    geocode_match_metadata json,
+    atd_mode_category_metadata json,
+    location_id character varying,
+    changes_approved_date timestamp without time zone,
+    austin_full_purpose character varying(1) DEFAULT 'N'::character varying NOT NULL,
+    original_city_id integer,
+    atd_fatality_count integer,
+    temp_record boolean DEFAULT false,
+    cr3_file_metadata jsonb,
+    cr3_ocr_extraction_date timestamp with time zone,
+    investigator_narrative_ocr text,
+    est_comp_cost_crash_based numeric(10,2) DEFAULT 0,
+    imported_at timestamp without time zone DEFAULT now()
```

Comment:
once again most of these changes aren't mine, the dump for this schema was just old!